Sources/Workspace/InitPackage.swift

```diff
@@ -621,23 +681,46 @@ public final class InitPackage {
 
     private func writeLibraryTestsFile(_ path: AbsolutePath) throws {
         try writePackageFile(path) { stream in
-            stream.send(
-                """
-                import XCTest
-                @testable import \(moduleName)
+            if options.supportedTestingLibraries.contains(.swiftTesting) {
+                stream.send("import Testing\n")
+            }
+            if options.supportedTestingLibraries.contains(.xctest) {
+                stream.send("import XCTest\n")
+            }
+            stream.send("@testable import \(moduleName)\n")
```

Comment:
While changing this code, would you mind accumulating this output in a mutable string or using string interpolation instead of `stream.send`? Under the hood `stream.send` does nothing more than string interpolation, but with an added layer of indirection that provides no benefits other than confusion. We didn't outright deprecate `stream.send` yet due to its widespread use in the early stages of SwiftPM development, but avoiding it should help us deprecating in the future.

```suggestion
            if options.supportedTestingLibraries.contains(.swiftTesting) {
                stream.send("import Testing\n")
            }
            if options.supportedTestingLibraries.contains(.xctest) {
                stream.send("import XCTest\n")
            }
            stream.send("@testable import \(moduleName)\n")
```

Reply:
I can certainly make a change here. What is the eventual destination of the buffered content? A single `stream.send()` call? Something else?

Reply:
Yeah, let's reduce this to a single `stream.send()` call for now. Thanks!